packages/notifications/examples/basic.md

```diff
@@ -0,0 +1,269 @@
+        Veggies es bonus vobis, proinde vos postulo essum magis kohlrabi welsh onion daikon amaranth
+        tatsoi tomatillo melon.
+        {state.showClose && (
+          <Close onClick={() => alert('closing alert')} aria-label="Close Alert" />
+        )}
+      </Alert>
+    </Col>
+  </Row>
+</Grid>;
+```
+
+### Notification
+
+The `<Paragraph>` component should be used to wrap multi-line content within a `<Notification>`.
+Otherwise, no wrapper is necessary. Usage of `<Close />` must contain an `aria-label` or other
+assistive technique to have discernible text.
+
+```jsx
+const { Field, Label, Radio, Toggle } = require('@zendeskgarden/react-forms/src');
+
+const { MD } = require('@zendeskgarden/react-typography/src');
+
+const StyledMd = styled(MD)`
+  ${props => `
+    color: ${props.theme.colors.foreground};
+    font-weight: ${props.theme.fontWeights.semibold}
+  `}
+`;
+
+const notificationTitles = {
+  success: 'Success',
+  error: 'Error',
+  warning: 'Warning',
+  info: 'Info'
+};
+
+const initialState = {
+  type: 'success',
+  isMultiLine: true
+};
+
+<Grid>
+  <Row>
+    <Col md={4}>
+      <Well isRecessed>
+        <div role="group" aria-label="alert types radio">
+          <StyledMd>Alert Types</StyledMd>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="success"
+              checked={state.type === 'success'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Success</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="error"
+              checked={state.type === 'error'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Error</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="warning"
+              checked={state.type === 'warning'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Warning</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="info"
+              checked={state.type === 'info'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Info</Label>
+            </Radio>
+          </Field>
+        </div>
+        <Field className="u-mt-xs">
+          <Toggle
+            checked={state.isMultiLine}
+            onChange={event => setState({ isMultiLine: event.target.checked })}
+          >
+            <Label>Show Multi-line</Label>
+          </Toggle>
+        </Field>
+      </Well>
+    </Col>
+    <Col md={8}>
+      <Notification type={state.type}>
```

Comment:
> By default in the sketch libraries, these are 300px wide. We don't spec a width for these in code, but at least showing them similarly to the alert example with the controls side-by-side will be helpful.

Currently, the `<Notification>` example renders to fit its parent container, similar to the current production example. It feels like an exact size would visually limit the different sizes that the `<Notification>` can be seen. Are there pain points around how `<Notification>` is currently presented in the production examples?

Reply:
Follow up: the first example from this PR had a `<Notification>` that took the full width of the screen. But now that it lives in the same row as the radio group, the widths of the `<Notification>` are good to go. Confirmed with @allisonacs in person.